tests/integration/sentinel/transitions/mark-for-outbound.spec.js

```diff
@@ -131,6 +137,103 @@ describe('mark_for_outbound', () => {
         });
     });
 
+    it('correctly creates and sends outbound request immediately with due "cron"', () => {
+      clock = sinon.useFakeTimers(new Date('2023-08-28T01:05:00'));
+      const report = makeReport();
+      const config = {
+        transitions: {
+          mark_for_outbound: true
+        },
+        outbound: {
+          test: {
+            relevant_to: 'doc.type === "data_record" && doc.form === "test"',
+            cron: '5 1 * * *',
+            destination: {
+              base_url: utils.hostURL(server.address().port),
+              path: WORKING_ENDPOINT
+            },
+            mapping: {
+              id: 'doc._id',
+              rev: 'doc._rev'
+            }
+          }
+        }
+      };
+
+      return utils
+        .updateSettings(config, 'sentinel')
+        .then(() => utils.saveDoc(report))
+        .then(() => sentinelUtils.waitForSentinel([report._id]))
+        .then(getTasks)
+        .then(tasks => {
+          expect(tasks).to.be.empty;
+        })
+        .then(() => utils.getDoc(report._id))
+        .then(report => {
+          expect(brokenEndpointRequests).to.be.empty;
+          expect(workingEndpointRequests).to.have.lengthOf(1);
+          expect(workingEndpointRequests[0]).to.deep.equal({
+            id: report._id,
+            rev: report._rev
+          });
+        })
+        .then(() => sentinelUtils.getInfoDoc(report._id))
+        .then(infoDoc => {
+          expect(infoDoc).to.nested.include({
+            type: 'info',
+            doc_id: report._id,
+            'completed_tasks[0].type': 'outbound',
+            'completed_tasks[0].name': 'test'
+          });
+        });
+    });
+
+    it('correctly skips outbound request immediately with not due "cron"', () => {
+      clock = sinon.useFakeTimers(new Date('2023-08-28T01:05:00'));
```

Comment:
I believe `sinon.useFakeTimers` won't work here. You're changing time in the test, not on the server code that actually runs. As far as I can tell, integration tests aren't actually running - the suite has been active for 4 hours so something's definitely wrong. 
I think you need a different approach here - maybe dynamically determine the cron value depending on server time at the moment of test running.